src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -57,7 +55,6 @@
 
   <PropertyGroup Condition="'$(TestWithCore)' != 'true' And '$(TestAppX)' != 'true'">
     <XunitExecutable Condition="'$(XunitExecutable)' == ''">xunit.console.exe</XunitExecutable>
```

Comment:
I searched and didn't find any projects that use non-core debugging in VS.  For testing purposes, I edited System.Collections.Concurrent.Tests.csproj to specify:
```
      <TargetGroup>netstandard1.3</TargetGroup>
      <OSGroup>Windows_NT</OSGroup>
      <TestTFM>netcore50</TestTFM>
```

After reloading the project, the StartProgram was set to `C:\Repos\corefx\bin/tests/Windows_NT.AnyCPU.Debug/System.Collections.Concurrent.Tests/netstandard1.3.netcore50/xunit.console.uwp.exe` and the StartArguments and StartWorkingDirectory were also appropriately set, so F5'ing successfully launched the UWP runner.

I'm not sure if putting those three properties in the csproj is how we should actually specify non-core debugging by default, but it worked as a hack for verifying these changes work with that scenario.